apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -262,7 +263,7 @@
             class="form-control pl-string-input-multiline"
             style="resize: none;"
             disabled
-            aria-describedby="pl-string-input-answer-{{uuid}}-suffix"
```

Comment:
Same comment, seems like there should be something for the label here and not just the suffix.

Reply:
The label here is fine:

```
    <label
        class="form-label"
        for="pl-string-input-{{uuid}}-area"
    >{{{label}}}</label>
    ```
    
    The `for` does the association.

Reply:
Though I added a fallback in f456869a190d52c527c4c169a14235ccbf00d45b.

Reply:
Relevant link: https://html-validate.org/rules/input-missing-label.html